docs/content.zh/docs/dev/datastream/operators/overview.md

```diff
@@ -755,3 +755,38 @@ some_stream.filter(...).slot_sharing_group("name")
 ```
 {{< /tab >}}
 {{< /tabs>}}
+
+## 名字和描述
+Flink里的算子和作业节点会有一个名字和一个描述。名字和描述。名字和描述都是用来介绍一个算子或者节点是在做什么操作，但是他们会被用在不同地方。
+
+名字会用在用户界面、线程名、日志、指标等场景。节点的名字会根据节点中算子的名字来构建。
+名字需要尽可能的简介，避免对外部系统产生大的压力。
+
+描述主要用在执行计划展示，以及用户界面展示。节点的描述同样是根据节点中算子的描述来构建。
+描述可以包括详细的算子行为的信息，以便我们在运行时进行debug分析。
+
+{{< tabs slotsharing >}}
```

Comment:
Do not use the same tabs name, otherwise it would  cause issues. Might change to `namedescription` ? Same to the English version